### ISSUE TITLE
Test app crashes on startup in an emulator

### ISSUE THREAD

**Issue by kmadsen** [mapbox/mapbox-navigation-android#4170]
**Android API:**
**Mapbox Navigation SDK version:**

I did a bisect to see where the issue happens. This is the first commit where the crash happens
https://github.com/mapbox/mapbox-navigation-android/commit/9aac182a9e094d276aa442bea2615e86feb71f33

```
    --------- beginning of crash
2021-03-23 17:34:35.856 4596-4596/com.mapbox.navigation.test_app E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.mapbox.navigation.test_app, PID: 4596
    java.lang.RuntimeException: Unable to start activity ComponentInfo{com.mapbox.navigation.test_app/com.mapbox.navigation.examples.core.MapboxNavigationActivity}: android.view.InflateException: Binary XML file line #8 in com.mapbox.navigation.test_app:layout/layout_activity_navigation: Binary XML file line #8 in com.mapbox.navigation.test_app:layout/layout_activity_navigation: Error inflating class com.mapbox.maps.MapView
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3449)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3601)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:85)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2066)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7656)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
     Caused by: android.view.InflateException: Binary XML file line #8 in com.mapbox.navigation.test_app:layout/layout_activity_navigation: Binary XML file line #8 in com.mapbox.navigation.test_app:layout/layout_activity_navigation: Error inflating class com.mapbox.maps.MapView
     Caused by: android.view.InflateException: Binary XML file line #8 in com.mapbox.navigation.test_app:layout/layout_activity_navigation: Error inflating class com.mapbox.maps.MapView
     Caused by: java.lang.reflect.InvocationTargetException
        at java.lang.reflect.Constructor.newInstance0(Native Method)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
        at android.view.LayoutInflater.createView(LayoutInflater.java:852)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1004)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:959)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:1121)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1082)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:680)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:532)
        at com.mapbox.navigation.examples.core.databinding.LayoutActivityNavigationBinding.inflate(LayoutActivityNavigationBinding.java:71)
        at com.mapbox.navigation.examples.core.databinding.LayoutActivityNavigationBinding.inflate(LayoutActivityNavigationBinding.java:65)
        at com.mapbox.navigation.examples.core.MapboxNavigationActivity.onCreate(MapboxNavigationActivity.kt:276)
        at android.app.Activity.performCreate(Activity.java:7995)
        at android.app.Activity.performCreate(Activity.java:7979)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1309)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3422)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3601)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:85)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2066)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7656)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
2021-03-23 17:34:35.856 4596-4596/com.mapbox.navigation.test_app E/AndroidRuntime: Caused by: java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "__ashldi3" referenced by "/data/app/~~EFH2OQTPeD38t4eaolkIBQ==/com.mapbox.navigation.test_app-bqpVPslkeccj12zKD9sgdQ==/lib/x86/libmapbox-maps.so"...
        at java.lang.Runtime.loadLibrary0(Runtime.java:1087)
        at java.lang.Runtime.loadLibrary0(Runtime.java:1008)
        at java.lang.System.loadLibrary(System.java:1664)
        at com.mapbox.common.loader.MapboxLibraryLoader.load(MapboxLibraryLoader.kt:19)
        at com.mapbox.maps.loader.MapboxMapStaticInitializer.loadMapboxMapNativeLib(MapboxMapStaticInitializer.java:12)
        at com.mapbox.maps.MapController.<clinit>(MapController.kt:367)
        at com.mapbox.maps.MapView.<init>(MapView.kt:111)
        at com.mapbox.maps.MapView.<init>(MapView.kt:84)
        at com.mapbox.maps.MapView.<init>(MapView.kt:66)
        at com.mapbox.maps.MapView.<init>(MapView.kt:60)
        	... 27 more
2021-03-23 17:34:35.869 4596-4630/com.mapbox.navigation.test_app E/FirebaseCrashlytics: Tried to write a fatal exception while no session was open.
```

**Comment by kmadsen** [comment]
The commit that caused this seems to be a map sdk version bump
https://github.com/mapbox/mapbox-navigation-android/commit/9aac182a9e094d276aa442bea2615e86feb71f33

```
-      mapboxMapSdk              : '10.0.0-beta.13',
+      mapboxMapSdk              : '10.0.0-beta.15',
```

Also the crash suggests it's something when loading the `/lib/x86/libmapbox-maps.so`

Caused by: java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "__ashldi3" referenced by "/data/app/~~EFH2OQTPeD38t4eaolkIBQ==/com.mapbox.navigation.test_app-bqpVPslkeccj12zKD9sgdQ==/lib/x86/libmapbox-maps.so"...

cc: @tobrun @LukasPaczos

**Comment by LukasPaczos** [comment]
Closing as a duplicate of https://github.com/mapbox/mapbox-maps-android/issues/163, this will be resolved when we upgrade to the Maps SDK `v10.0.0-beta.16` this week.
